mpas_analysis/ocean/meridional_heat_transport.py

```diff
@@ -0,0 +1,213 @@
+    # Latitude is from binBoundaryMerHeatTrans written in
+    #  mpaso.hist.am.meridionalHeatTransport.*.nc
+    # Depth is from refZMid, also in mpaso.hist.am.meridionalHeatTransport.*.nc
+    try:
+        mhtFile = historyStreams.readpath('meridionalHeatTransportOutput')[0]
+    except ValueError:
+        raise IOError('No MPAS-O MHT history file found: need at least one ')
+
+    print '  Read in depth and latitude...'
+    ncFile = netCDF4.Dataset(mhtFile, mode='r')
+    # reference depth [m]
+    refZMid = ncFile.variables['refZMid'][:]
+    binBoundaryMerHeatTrans = ncFile.variables['binBoundaryMerHeatTrans'][:]
+    binBoundaryMerHeatTrans = binBoundaryMerHeatTrans*rad_to_deg
+    ncFile.close()
+
+    ########################################################################
+    # Mark P Note: Currently only supports global MHT.
+    # Need to add variables merHeatTransLatRegion and merHeatTransLatZRegion
+    # these are not computed by default in ACME right now.
+    # Then we will need to add another section for regions with a loop
+    # over number of regions.
+    ########################################################################
+    variableList = ['avgMeridionalHeatTransportLat',
+                    'avgMeridionalHeatTransportLatZ']
+
+    print '\n  Compute and plot global meridional heat transport'
+
+    outputDirectory = build_config_full_path(config, 'output',
+                                             'mpasClimatologySubdirectory')
+
+    make_directories(outputDirectory)
+
+    outputFileClimo = '{}/meridionalHeatTransport_years{:04d}-{:04d}.nc'.format(
+                       outputDirectory, dictClimo['startYearClimo'],
+                       dictClimo['endYearClimo'])
+    if not os.path.exists(outputFileClimo):
+        print '   Load data...'
+        ds = open_multifile_dataset(
+            fileNames=dictClimo['inputFilesClimo'],
+            calendar=calendar,
+            config=config,
+            simulationStartTime=simulationStartTime,
+            timeVariableName='Time',
+            variableList=variableList,
+            variableMap=variableMap,
+            startDate=dictClimo['startDateClimo'],
+            endDate=dictClimo['endDateClimo'])
+
+        changed, startYear, endYear = \
+            climatology.update_start_end_year(ds, config, calendar)
+        if changed:
+            # update the file name in case the start and end years changed
+            outputFileClimo = \
+                '{}/meridionalHeatTransport_years{:04d}-{:04d}.nc'.format(
+                    outputDirectory, startYear, endYear)
+
+            dictClimo['startYearClimo'] = startYear
+            dictClimo['endYearClimo'] = endYear
+
+        # Compute annual climatology
+        annualClimatology = \
+            climatology.compute_climatology(ds,
+                                            monthDictionary['ANN'],
+                                            calendar)
+
```

Comment:
I think the else case belongs here:
```python
else:
     annualClimatology = xr.open_dataset(outputFileClimo)
```
And the remaining code should happen either way.

In fact, I think the `if` case and the else case I suggested above should be switched, because I never like to have an if/else that is actually and `if not`/`else`.